src/Components/GrowthChartSelector/ChartSelector.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Menu } from '@headlessui/react';
+import { CategoryCodes } from '../../types/chartDataTypes';
+import { chartData } from '../../DataSets/WhoStandardDataSets/ZScores/ChartDataZscores';
+
+interface ChartSelectorProps {
+    category: keyof typeof CategoryCodes;
+    dataset: keyof typeof chartData;
+    setCategory: (category: keyof typeof CategoryCodes) => void;
+    setDataset: (dataset: keyof typeof chartData) => void;
+}
+
+export const ChartSelector = ({
+    category,
+    dataset,
+    setCategory,
+    setDataset,
+}: ChartSelectorProps) => {
+    const handleCategoryChange = (key: string) => {
+        const newCategory = CategoryCodes[key as keyof typeof CategoryCodes];
+        setCategory(newCategory);
+        setDataset(Object.keys(chartData[newCategory].datasets)[0]);
+    };
+
+    const handleDatasetChange = (key: string) => {
+        const newDataset = key as keyof typeof chartData;
+        setDataset(newDataset);
+    };
+
+    return (
+        <div className='w-full flex'>
+            <div className='flex gap-2 text-sm pl-12'>
+                <div className='flex flex-col relative min-w-fit'>
+                    <Menu>
+                        {({ open }) => (
+                            <>
+                                <Menu.Button
```

Comment:
On the div above, may you try to split the alternatives with a small line? Me personally think that would be a nice way to seperate the different alternatives in a clear way